implementations/rust/ockam/ockam_api/Cargo.toml

```diff
@@ -43,7 +43,7 @@ kafka-protocol = "0.6.1"
 lru = "0.11.1"
 miette = "5.10.0"
 minicbor = { version = "0.19.0", features = ["alloc", "derive"] }
-nix = "0.27"
+nix = {verion = "0.27", features = ["signal"] }
```

Comment:
There's a typo here, `verion`.

Reply:
Bummer!

Reply:
Should be fixed now 🤞

Reply:
I built locally before and it (seemed to) work(ed). Not sure what happened. There are lots of crates, though, and some might not be built when just using `cargo check` form the workspace directory. Anyway, this time I actually built that particular crate, and even tested it: `cargo test -p ockam_api`